ios/REAEventDispatcher.m

```diff
@@ -1,12 +1,22 @@
+#ifndef RCT_NEW_ARCH_ENABLED
+
 #import <RNReanimated/REAEventDispatcher.h>
+#import <RNReanimated/REAModule.h>
 #import <React/RCTBridge+Private.h>
 #import <React/RCTDefines.h>
 
 @implementation REAEventDispatcher
 
 - (void)sendEvent:(id<RCTEvent>)event
 {
-  [[[self bridge] moduleForName:@"ReanimatedModule"] eventDispatcherWillDispatchEvent:event];
+  __weak static REAModule *reaModule;
+  __weak static RCTBridge *lastBridge;
```

Comment:
What about retain cycle? Can we keep refference to bridge?

Reply:
You are right I missed it. 👍 

Just to be sure - did you tested those changes with continues events (for example scrolling) and app reload?

Reply:
Nope, not yet, but I will, thanks! Honestly, I'm a bit afraid about this change, not sure if the performance gain is worth it.

Reply:
Reloading while scrolling works fine, tested on ScrollViewExample.tsx.